torch_ort_inference/torch_ort/ortinferencemodule/_custom_op_symbolic_registry.py

```diff
@@ -0,0 +1,51 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import torch
+from packaging.version import Version
+from torch.onnx import register_custom_op_symbolic
+
+class CustomOpSymbolicRegistry:
+    _SYMBOLICS = {}
+
+    @classmethod
+    def register(cls, name, domain, fn):
+        cls._SYMBOLICS[domain + "::" + name] = fn
+
+    @classmethod
+    def register_all(cls):
+        for name, fn in cls._SYMBOLICS.items():
+            # Symbolic name is in format: domain::name
+            # Exporter will fail to register symbolic with non-empty domain when torch version is < 1.11.0.
+            if Version(torch.__version__) >= Version("1.11.0") or name.startswith("::"):
+                register_custom_op_symbolic(name, fn, 1)
+
+
+def register_symbolic(name, domain=""):
+    def symbolic_wrapper(fn):
+        CustomOpSymbolicRegistry.register(name, domain, fn)
+        return fn
+
+    return symbolic_wrapper
+
+# Unsupported Aten ops to be added here
+
```

Comment:
what is the reason we are adding the custom symblics? Are all these ops being exported as `Aten` ops? Is that expected?

Reply:
+1 - All these ops are available in ONNX

Reply:
@askhade and I looked into what might be going on behind the scenes leading to these ops being exported as Aten to begin with. It seems like the `opset_version` being used by the exporter is 13 (default since it is not provided in the `export` method from torch_ort_inference code) as can be seen [here](https://github.com/pytorch/ort/blob/89f86492bd6bfe3b0ace3c0f1643ed3dcd427256/torch_ort_inference/torch_ort/ortinferencemodule/ortinferencemodule.py#L155-L161).

If the `opset_version` is provided (I recommend setting it to 15 or above), these ops will be exported as onnx ops (thereby removing the need to register custom symbolics).

Reply:
@baijumeswani, updated the PR with opset version changes and removed the obsolete custom symbolics code.